backend/corpora/common/utils/cxg_constants.py

```diff
@@ -1,4 +1,4 @@
 class CxgConstants(object):
     # The CXG container version number.  Must be a semver string (major.minor.patch)
     # DO NOT UPDATE THIS WITHOUT ALSO UPDATING CXG SPECIFICATION.
-    CXG_VERSION = "0.2.0"
+    CXG_VERSION = "0.3.0"
```

Comment:
question: is this meant to coincide with [this cellxgene schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md)? If so, we're actually on 2.0.0 currently and all 3.0.0 changes are in a feature branch. Not sure where the disconnect originated with using 0.2 instead of 2.0, but please let me know if this is meant to refer to something else

Reply:
I'm not 100% sure, but I think there's the schema version and the CXG version, which are different.

Reply:
Minor, and not related to this PR but I did notice that you updated the cxg spec doc but I think we should also update the current version heading at the top from 0.2.0 -> 0.3.0 (+ the date) for clarity: https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs/cxg.md#cxg-data-format-specification

Reply:
> I'm not 100% sure, but I think there's the schema version and the CXG version, which are different.
correct, they're different